src/components/common/ExternalLink/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { OpenInNewRounded } from '@mui/icons-material'
+import { Box, Link, type LinkProps } from '@mui/material'
+
+/**
+ * Renders an external Link which always sets the noopener and noreferrer rel attribute and the target to _blank.
+ * It also always adds the external link icon as end adornment.
+ */
+const ExternalLink = ({
+  suppressIcon = false,
+  children,
+  ...props
+}: Omit<LinkProps, 'target' | 'rel'> & { suppressIcon?: boolean }) => {
+  return (
+    <Link rel="noreferrer noopener" target="_blank" {...props}>
+      <Box display="inline-flex" alignItems="center" gap={0.2}>
+        {children}
+        {!suppressIcon && <OpenInNewRounded fontSize="small" />}
+      </Box>
+    </Link>
```

Comment:
I'd suggest we just give this the normal `LinkProps` by default and spread all them all on `Link` so that it can be granularly used.

Do we more often want to show the icon? It's personal taste, but I'd probably invert the icon-related prop and just have `icon` or something similar.